swap/src/monero.rs

```diff
@@ -360,6 +374,27 @@ mod tests {
         );
     }
 
+    #[test]
+    fn geting_max_bitcoin_to_trade() {
+        let monero = "10";
+        let amount = Amount::parse_monero(monero).unwrap();
+        let bitcoin_price_sats = bitcoin::Amount::from_sat(382_900);
+        let monero_max_from_bitcoin = amount.max_bitcoin_for_price(bitcoin_price_sats).unwrap();
+        assert_eq!(
+            bitcoin::Amount::from_sat(3_828_988),
+            monero_max_from_bitcoin
+        );
```

Comment:
A code style nit:

Have you heard of "arrange - act - assert" when it comes to testing? It makes tests heaps more readable when we strive to:

- Have a single statement per block (arrange/act/assert) or at least minimise their size
- Use whitespace to separate those blocks

For example:


```suggestion
        let amount = Amount::parse_monero("10").unwrap();
        let bitcoin_price_sats = bitcoin::Amount::from_sat(382_900);

        let monero_max_from_bitcoin = amount.max_bitcoin_for_price(bitcoin_price_sats).unwrap();

        assert_eq!(
            bitcoin::Amount::from_sat(3_828_988),
            monero_max_from_bitcoin
        );
```

Reply:
No! actually I didn't I thought that the normal thing was to call the function inside the Assertion!. Thank you for bringing that up!

Reply:
I wouldn't call the function inside the assertion, that would merge the "act" and "assert" stage :)

Simply adding whitespace between them makes it visually easy to identify the relevant parts of the test.

Reply:
just changed the tests